dashboard/app/helpers/levels_helper.rb

```diff
@@ -475,7 +475,7 @@ def azure_speech_service_options
   end
 
   def disallowed_html_tags
-    DCDO.get('disallowed_html_tags', ['script', 'iframe'])
+    DCDO.get('disallowed_html_tags', [])
```

Comment:
note about this change: i was previously keeping the defaults for this DCDO flag up-to-date with production, but i want to change it for a few reasons:
1. keeping this list up-to-date is becoming cumbersome as we add more tags, and the whole point of using a DCDO flag is that it doesn't require a code change
2. it will cause confusion if the value returned from DCDO is different from this default set, so this value should act more like a fallback than it currently does

however, this flag requires some knowledge for how to use it, so a follow-up task for me this sprint ([STAR-1535](https://codedotorg.atlassian.net/browse/STAR-1535)) is to write some general documentation for WebLab and include an explanation for this flag.